server/app/services/question/types/AddressQuestionDefinition.java

```diff
@@ -66,7 +63,13 @@ ValidationPredicates getDefaultValidationPredicates() {
     return AddressValidationPredicates.create();
   }
 
+  @JsonIgnore
```

Comment:
Validation predicates are included in the json from the config, so we don't need to include the fields as well.